status/mware.go

```diff
@@ -19,14 +19,14 @@ func InterceptSdk(sdkId string, reporter Reporter, transport http.RoundTripper)
 func (i *clientInterceptor) RoundTrip(r *http.Request) (*http.Response, error) {
 	resp, err := i.RoundTripper.RoundTrip(r)
 	if err != nil {
-		i.reporter.ReportError(i.sdkId, err)
+		i.reporter.ReportError(i.sdkId, "config fetch failed")
```

Comment:
I'm not sure if it is a good approach to "swallow" the error - debug reasons.

Reply:
It won't swallow; the function returns the error for further processing (the SDK will log it eventually). This part just won't send the error to show on the status page (which is public).